ext/snmp/snmp.stub.php

```diff
@@ -2,49 +2,70 @@
 
 /** @alias snmprealwalk */
-function snmpwalkoid(string $host, string $community, array|string $object_id, int $timeout = -1, int $retries = -1): array|bool {}
+function snmpwalkoid(string $hostname, string $community, array|string $object_id, int $timeout = -1, int $retries = -1): array|bool {}
 
-function snmpset(string $host, string $community, array|string $object_id, array|string $type, array|string $value, int $timeout = -1, int $retries = -1): array|bool {}
+function snmpset(string $hostname, string $community, array|string $object_id, array|string $type, array|string $value, int $timeout = -1, int $retries = -1): array|bool {}
 
 function snmp_get_quick_print(): bool {}
 
-function snmp_set_quick_print(bool $quick_print): bool {}
+function snmp_set_quick_print(bool $enable): bool {}
 
-function snmp_set_enum_print(bool $enum_print): bool {}
+function snmp_set_enum_print(bool $enable): bool {}
 
 function snmp_set_oid_output_format(int $oid_format): bool {}
 
 /** @alias snmp_set_oid_output_format */
 function snmp_set_oid_numeric_print(int $oid_format): bool {}
 
-function snmp2_get(string $host, string $community, array|string $object_id, int $timeout = -1, int $retries = -1): stdClass|array|string|bool {}
+function snmp2_get(string $hostname, string $community, array|string $object_id, int $timeout = -1, int $retries = -1): stdClass|array|string|bool {}
 
-function snmp2_getnext(string $host, string $community, array|string $object_id, int $timeout = -1, int $retries = -1): stdClass|array|string|bool {}
+function snmp2_getnext(string $hostname, string $community, array|string $object_id, int $timeout = -1, int $retries = -1): stdClass|array|string|bool {}
 
-function snmp2_walk(string $host, string $community, array|string $object_id, int $timeout = -1, int $retries = -1): array|bool {}
+function snmp2_walk(string $hostname, string $community, array|string $object_id, int $timeout = -1, int $retries = -1): array|bool {}
 
-function snmp2_real_walk(string $host, string $community, array|string $object_id, int $timeout = -1, int $retries = -1): array|bool {}
+function snmp2_real_walk(string $hostname, string $community, array|string $object_id, int $timeout = -1, int $retries = -1): array|bool {}
 
-function snmp2_set(string $host, string $community, array|string $object_id, array|string $type, array|string $value, int $timeout = -1, int $retries = -1): array|bool {}
+function snmp2_set(string $hostname, string $community, array|string $object_id, array|string $type, array|string $value, int $timeout = -1, int $retries = -1): array|bool {}
 
-function snmp3_get(string $host, string $sec_name, string $sec_level, string $auth_protocol, string $auth_passphrase, string $priv_protocol, string $priv_passphrase, array|string $object_id, int $timeout = -1, int $retries = -1): stdClass|array|string|bool {}
+function snmp3_get(
+    string $hostname, string $sec_name, string $sec_level,
```

Comment:
```suggestion
    string $hostname, string $security_name, string $security_level,
```

Reply:
Heh, I knew this was going to happen :) I got a bit stingy with characters because the signature is soooo long...